fastlane/lib/fastlane/actions/ensure_xcode_version.rb

```diff
@@ -1,9 +1,5 @@
 module Fastlane
   module Actions
-    module SharedValues
-      ENSURE_XCODE_VERSION_CUSTOM_VALUE = :ENSURE_XCODE_VERSION_CUSTOM_VALUE
```

Comment:
unused?

Reply:
Indeed, couldn't find any usage of it:

![image](https://user-images.githubusercontent.com/10795657/56344975-f85c7480-61be-11e9-8d08-5dc5961f86f4.png)

Reply:
Ok, so these are leftovers. Did you investigate when or how those got orphaned? Always seems strange when something like this is in the codebase.